explorer/src/components/SupplyCard.tsx

```diff
@@ -66,7 +66,9 @@ const renderHeader = () => {
     <div className="card-header">
       <div className="row align-items-center">
         <div className="col">
-          <h4 className="card-header-title">Supply Overview</h4>
+          <h4 id="overview" className="card-header-title">
+            <a href="#overview">Supply Overview</a>
```

Comment:
I made a new bulletproof system for scrolling to anchors! Check it out in
#24861.

Reply:
Essentially it's this:

```
import { useScrollAnchor } from "providers/scroll-anchor";
/* ... */
const scrollAnchorRef = useScrollAnchor('overview');
/* ... */
<h4 className="card-header-title" ref={scrollAnchorRef}>
  Supply Overview
</h4>
```